README.md

```diff
@@ -1,5 +1,4 @@
-# Tailwindcss-vertical-rhythm
-
+# Tailwindcss-vertical-rhythm ![NPM](https://img.shields.io/npm/l/tailwindcss-vertical-rhythm)    ![npm minified_bundle size](https://badgen.net/bundlephobia/min/lodash) ![npm minzipped_bundle size](https://img.shields.io/bundlephobia/minzip/lodash?color=%2300FF00)
```

Comment:
Hey @sudhansukinng  thanks for this, it looks like you're linking to lodash no the plugin for bundle size? Also the MIT license alt tag says NPM.